### PR TITLE
CI Refactor

### PR DESCRIPTION
**Pull request by romansertsov**
CI Refactor

**master** branch need to be renamed to **main**

### CODE REVIEW

File: ci/run-integration-tests.ps1 [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    [Parameter(Mandatory=$true)]
+    [string]$RepoName
+)
+
+Push-Location $RepoName
+
+try
+{
+    Write-Output "Running integration tests"
+    $env:RESOURCE_KEY = $Options.Keys.TestResourceKey
```
Comment (ben51degrees) [inline ci/run-integration-tests.ps1]: Options isn't actually named as a parameter in this script. I didn't spot this in device-detection and pipeline, but could you double check?

File: tests/geoLocationCloud.test.js [hunk 2]
```diff
@@ -107,19 +107,19 @@ if (isAsync) {
...
-
-    const pipeline = new GeoLocation.GeoLocationPipelineBuilder({
-      resourceKey: myResourceKey,
-      locationProvider: 'digitalelement'
-    }).build();
-    var engine = pipeline.flowElements.location_digitalelement;
-
-    await testAvailableProperties(done, pipeline, engine);
-  });
+  // test('Available Properties Digital Element', async done => {
```
Comment (ben51degrees) [inline tests/geoLocationCloud.test.js]: The two tests commented out here can actually be removed completely now, as that provider is no longer supported.

**Review by ben51degrees — CHANGES_REQUESTED** [review]

File: ci/run-integration-tests.ps1 (hunk 1 above)
Comment (romansertsov) [reply ci/run-integration-tests.ps1]: Note: Actually it's kinda weird that was working just fine, i'll change add $Keys param to make it more obvious
i think $Options is something similar to ...args in js

Comment (ben51degrees) [reply ci/run-integration-tests.ps1]: Maybe it's available in the frame above, and is accessible here too. But the change you made is better.

**Review by ben51degrees — APPROVED** [review]

**Pull request merged**